pkg/controller/pravega/pravega_controller.go

```diff
@@ -206,8 +208,53 @@ func makeControllerPodSpec(p *api.PravegaCluster) *corev1.PodSpec {
 	if p.Spec.Pravega.ControllerSecurityContext != nil {
 		podSpec.SecurityContext = p.Spec.Pravega.ControllerSecurityContext
 	}
+	podSpec.InitContainers = []corev1.Container{}
 	if p.Spec.Pravega.ControllerInitContainers != nil {
-		podSpec.InitContainers = p.Spec.Pravega.ControllerInitContainers
+		podSpec.InitContainers = append(podSpec.InitContainers, p.Spec.Pravega.ControllerInitContainers...)
```

Comment:
Say I am writing my own initcontainer and I add in there a command to copy some things in a volumeMount path.   But I'd have to refer to a volume.  That volume is at the pod level.   That volume is only created in the case of the AuthImplementations, atm.

Does that mean the generic initcontainers as they are right now, would be essentially useless?  (from the pov of copying something in the main container I mean.  I suppose init containers can have other uses as well)

Reply:
Yes, in addition to init containers, we have to create empty dir volume. This can be achieved by providing `emptyDirVolumeMounts` in pravega options